lib/src/view/coordinate_training/coordinate_training_screen.dart

```diff
@@ -351,61 +404,54 @@ class _SettingsState extends ConsumerState<_Settings> {
     final trainingPrefs = ref.watch(coordinateTrainingPreferencesProvider);
 
     return Column(
-      mainAxisSize: MainAxisSize.min,
-      mainAxisAlignment: MainAxisAlignment.center,
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      crossAxisAlignment: CrossAxisAlignment.center,
       children: [
-        PlatformListTile(
-          title: Text(context.l10n.side),
-          trailing: Padding(
-            padding: Styles.horizontalBodyPadding,
-            child: Wrap(
-              spacing: 8.0,
-              children: SideChoice.values.map((choice) {
-                return ChoiceChip(
-                  label: Text(sideChoiceL10n(context, choice)),
-                  selected: trainingPrefs.sideChoice == choice,
-                  showCheckmark: false,
-                  onSelected: (selected) {
-                    widget.onSideChoiceSelected(choice);
-                    ref
-                        .read(coordinateTrainingPreferencesProvider.notifier)
-                        .setSideChoice(choice);
-                  },
-                );
-              }).toList(),
-            ),
-          ),
+        Wrap(
+          spacing: 8.0,
+          children: SideChoice.values.map((choice) {
```

Comment:
While you're at it, I think we could use the `Filter` class here that I made public in https://github.com/lichess-org/mobile/pull/1000

Reply:
The problem is that I removed the text because I think it looks better without it and the `Filter` widget requires it.

I'll post a before/after video so you can see what I mean.

Reply:
Ah got it 👍 (impressive speed btw)